<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProjectDialog</class>
 <widget class="QDialog" name="NewProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create New Project</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Project Name</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>projectNameLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="projectNameLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;Description</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>projectDescriptionLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="projectDescriptionLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&amp;Location</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>locationLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="locationLineEdit"/>
       </item>
       <item>
        <widget class="QToolButton" name="locationToolButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Include Paths</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QToolButton" name="addIncludePathToolButton">
          <property name="toolTip">
           <string>Add Include Path</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="asproject.qrc">
            <normaloff>:/images/inc_add.png</normaloff>:/images/inc_add.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="delIncludePathToolButton">
          <property name="toolTip">
           <string>Remove Include Path</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="asproject.qrc">
            <normaloff>:/images/inc_del.png</normaloff>:/images/inc_del.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="includePathsListWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>System Include Paths</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QToolButton" name="addSystemIncludePathToolButton">
          <property name="toolTip">
           <string>Add System Include Path</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="asproject.qrc">
            <normaloff>:/images/inc_add.png</normaloff>:/images/inc_add.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="delSystemIncludePathToolButton">
          <property name="toolTip">
           <string>Remove System Include Path</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="asproject.qrc">
            <normaloff>:/images/inc_del.png</normaloff>:/images/inc_del.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="importIncludePathToolButton">
          <property name="toolTip">
           <string>Import System Include Path From Other Project</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="asproject.qrc">
            <normaloff>:/images/env_import.png</normaloff>:/images/env_import.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="systemIncludePathsListWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Defines</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QToolButton" name="addDefineToolButton">
          <property name="toolTip">
           <string>Add Project Define</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="asproject.qrc">
            <normaloff>:/images/inc_add.png</normaloff>:/images/inc_add.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="delDefineToolButton">
          <property name="toolTip">
           <string>Remove Project Define</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="asproject.qrc">
            <normaloff>:/images/inc_del.png</normaloff>:/images/inc_del.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="definesListWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Precompiled Header</string>
       </property>
       <property name="buddy">
        <cstring>precompiledHeaderLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="precompiledHeaderLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okPushButton">
       <property name="text">
        <string>Create</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="asproject.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>NewProjectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okPushButton</sender>
   <signal>clicked()</signal>
   <receiver>NewProjectDialog</receiver>
   <slot>tryAccept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>locationToolButton</sender>
   <signal>clicked()</signal>
   <receiver>NewProjectDialog</receiver>
   <slot>choiseLocation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>19</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addIncludePathToolButton</sender>
   <signal>clicked()</signal>
   <receiver>NewProjectDialog</receiver>
   <slot>addIncludePath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delIncludePathToolButton</sender>
   <signal>clicked()</signal>
   <receiver>NewProjectDialog</receiver>
   <slot>delIncludePath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addSystemIncludePathToolButton</sender>
   <signal>clicked()</signal>
   <receiver>NewProjectDialog</receiver>
   <slot>addSystemIncludePath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delSystemIncludePathToolButton</sender>
   <signal>clicked()</signal>
   <receiver>NewProjectDialog</receiver>
   <slot>delSystemIncludePath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addDefineToolButton</sender>
   <signal>clicked()</signal>
   <receiver>NewProjectDialog</receiver>
   <slot>addDefine()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delDefineToolButton</sender>
   <signal>clicked()</signal>
   <receiver>NewProjectDialog</receiver>
   <slot>delDefine()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>9</x>
     <y>493</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>tryAccept()</slot>
  <slot>choiseLocation()</slot>
  <slot>addIncludePath()</slot>
  <slot>delIncludePath()</slot>
  <slot>addSystemIncludePath()</slot>
  <slot>delSystemIncludePath()</slot>
  <slot>addDefine()</slot>
  <slot>delDefine()</slot>
 </slots>
</ui>
